src/stylesheets/components/_concept.scss

```diff
@@ -0,0 +1,37 @@
+.concept {
+  background: linear-gradient($florence-blue, $florence-light-blue);
+  color: $white;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-top: 180px;
+  padding-top: 130px;
+  position: relative;
+  width: 100%;
+  
+  .section-title {
+  	color: $white;
+  	margin-bottom:-10px;
+  }
+
+  .section-intro {
+  	color: $white;
+  	font-size: 2.5em;
+  }
+
+  p {
+    margin: 10px auto 30px auto;
```

Comment:
You could remove the last `auto` here, as it'll be repeated from the other instance, e.g. `margin: 10px auto 30px;`